apps/core/tests.py

```
@@ -13,6 +16,43 @@ class BaseTestCase(TestCase):
 
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.mock_storage()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        # Disable storage override
+        try:
+            cls._storage_override.disable()
+        except Exception:
+            pass
```

Ruff (0.14.7)

`try`-`except`-`pass` detected, consider logging the exception

(S110)